src/components/footer/Footer.component.tsx

```diff
@@ -0,0 +1,36 @@
+import Facebook from '@/assets/fb.png';
+import Instagram from '@/assets/insta.png';
+import Linkedin from '@/assets/linkedin.png';
+
+import styles from './footer.module.css';
+
+export const FooterComponent = () => (
+    <footer className={styles.footer}>
+        <div className={styles.footer__content}>
+            <ul className={styles.footer__links}>
+                <li>
+                    <a className={styles.footer__link} href="https://uk-ua.facebook.com/" target="_blank" rel="noreferrer">
```

Comment:
Please add a hove states for social links and link to the course page - [design](https://www.figma.com/file/mRAOMdJiAsD8fHSVnYwUl9/React-web-page?type=design&node-id=1-13005&mode=design&t=nyqcP6EXfnyRZdaC-0)
![image](https://github.com/demasha0908/fe-react-2024/assets/98031768/b81706e4-b361-44ff-824d-1120c0266f6a)